test/cluster/test_tablet_snapshot.py

```
@@ -0,0 +1,215 @@
+    for host in topo.all_hosts():
+        assert host.dc == "dc1"
+        assert host.rack is not None
+
+
+def take_manual_snapshot(host: str, workdir: Path, output_dir: str = "manual_snapshot") -> Path:
+    """
+    Captures a snapshot by running the cqlsh commands --manual prints, as a user without
+    direct cluster access would, and returns the directory they wrote.
+    """
+    snapshot_script = TOP_SRC_DIR / "scripts" / "tablets" / "snapshot.py"
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)